compiler/test/formatter_outputs/application.gr

```diff
@@ -6,7 +6,10 @@ let data = [>]
 
 let busNumbers = Array.map(
   ((num, idx)) => {
-    (Option.expect("Expected a number", None), idx)
+    (
```

Comment:
Did this really need to break? That looks like it's all within length.

Reply:
Fixed.